script/test

```diff
@@ -74,10 +74,10 @@ get_openthread()
 {
     if ! [[ -x $OT_DIR ]]; then
         mkdir -p "$(dirname "$OT_DIR")"
-        git clone https://github.com/openthread/openthread.git "$OT_DIR"
+        git clone https://github.com/openthread/openthread.git "$OT_DIR" --branch master --depth 1
         (
             cd "$OT_DIR"
-            ./script/bootstrap
+            repeat 3 ./script/bootstrap
```

Comment:
Have you ever tried just remove `./script/bootstrap`? In my experience of writing integration tests for OT Commissioner, simulation binaries can be built without bootstrapping (it is mainly installing embedded toolchains and code formatting tool in the script).

Reply:
I will try that

Reply:
I removed `./script/boostrap` and tests failed because tools like `autoconf` and `automake` is missing. 
I don't want to over-complicate this test script by installing just these tools by itself.
I will leave `repeat 3 ./script/bootstrap` unchanged.